tests/functional/c/consider/consider_iterating_dictionary.txt

```diff
@@ -1,15 +1,17 @@
-consider-iterating-dictionary:23:16::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:24:16::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:25:16::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:26:21::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:27:24::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:28:24::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:29:24::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:30:29::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:31:11::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:36:24::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:36:55::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:37:31::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:37:61::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:38:30::Consider iterating the dictionary directly instead of calling .keys()
-consider-iterating-dictionary:38:60::Consider iterating the dictionary directly instead of calling .keys()
+consider-iterating-dictionary:23:16::Consider iterating the dictionary directly instead of calling .keys():HIGH
```

Comment:
Sorry for the large diff. I used the command to automatically update the data. Want me to change this?

Reply:
Doesn't really matter. At least not here. With 200+ changes, I would probably evaluate that differently.